src/cli/peakdetector/peakdetector.pro

```diff
@@ -21,7 +22,8 @@ INCLUDEPATH +=  $$top_srcdir/src/core/libmaven     \
                 $$top_srcdir/3rdparty/libcdfread   \
                 $$top_srcdir/src/pollyCLI          \
                 $$top_srcdir/3rdparty/obiwarp      \
-                $$top_srcdir/3rdparty/Eigen
+                $$top_srcdir/3rdparty/Eigen        \
+                $$top_srcdir/src/gui
```

Comment:
@rish9511 Is it a good idea to give access to src/gui folder in cli?

Reply:
We should have a folder called "common" ideally.
We plan to change it